spinalcordtoolbox/vertebrae/core.py

```diff
@@ -312,21 +315,30 @@ def clean_labeled_segmentation(fname_labeled_seg, fname_seg, fname_labeled_seg_n
     """
     # remove voxels in segmentation_labeled that are not in segmentation
     img_labeled_seg = Image(fname_labeled_seg)
+    img_labeled_seg.change_orientation('RPI')
+    native_or = img_labeled_seg.orientation
     img_seg = Image(fname_seg)
+    img_seg.change_orientation('RPI')
     data_labeled_seg_mul = img_labeled_seg.data * img_seg.data
-    # dilate to add voxels in segmentation that are not in segmentation_labeled
-    data_labeled_seg_dil = dilate(img_labeled_seg.data, 2, 'ball')
+
+    # locate 'holes' in the labeled spinal cord segmentation
```

Comment:
```suggestion
    # Locate voxels that are in the native cord segmentation but not in the output labeled segmentation.
    # We obtain the index of these voxels, and they will later be filled with the appropriate label value.
```

Reply:
I changed to code of `clean_labeled_segmentation` back to the original since we are creating the flag.